changelog.dd

```diff
@@ -28,12 +28,12 @@ $(BUGSTITLE Library Changes,
         exposed))
     $(LI $(RELATIVE_LINK2 iota-length-size_t, `std.range.iota's `.length`
         property is fixed to `size_t` instead of the type being iterated))
-    $(LI $(XREF uni, isNumber) and $(XREF uni, isPunctuation) now use a separate,
+    $(LI $(REF isNumber, std,uni) and $(REF isPunctuation, std,uni) now use a separate,
```

Comment:
I guess we don't want to force people to clone dlang.org just to check if their changelog entry is ok. I don't like duplicating macros here in changelog.dd, because the ones from dlang.org are used for the actual changelog. Keeping them in sync is going to be annoying.

Would it be acceptable if we built the project changelogs with dlang.org, and rely on the auto-tester for checking? I mean, that's what people have to do with other documentation changes, isn't it? Would also help reviewers.